src/notifications/notificationmanager.cpp

```diff
@@ -103,6 +105,9 @@ NotificationManager::NotificationManager(QObject *parent) :
     databaseCommitTimer.setSingleShot(true);
     connect(&databaseCommitTimer, SIGNAL(timeout()), this, SLOT(commit()));
 
+    expirationTimer.setSingleShot(true);
+    connect(&expirationTimer, SIGNAL(timeout()), this, SLOT(expire()));
+
```

Comment:
Probably not critical here, regular QTimer will stop when device goes to deep sleep.